eng/Version.Details.xml

```diff
@@ -83,14 +83,6 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>6f33f02efeafb578b3f99da08cb2670e0470c917</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Net.Compilers" Version="3.0.0-beta4-final">
```

Comment:
If we want to have auto updates we need to add the package here, but from looking at your changes we're just going to consume arcade's defined version?

Reply:
You don't need the version here to get auto updates. The updates flow thru Arcade.

Reply:
@safern, the version flows from arcade, which does have the subscription. I wouldn't expect we need to set it in CoreFX as well.

Reply:
That is, you'll get new compilers with new Arcade version.